libs/domain/product/availability/availability.component.ts

```diff
@@ -0,0 +1,80 @@
+import { ContentMixin, defaultOptions } from '@spryker-oryx/experience';
+import { ProductMixin } from '@spryker-oryx/product';
+import { AlertType } from '@spryker-oryx/ui';
+import { computed, hydratable, i18n } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import {
+  CartItemAvailabilityOptions,
+  StockAvailability,
+} from './availability.model';
+import { availabilityStyles } from './availability.styles';
+
+@hydratable()
+@defaultOptions({ threshold: 5, enableIndicator: true })
+export class ProductAvailabilityComponent extends ProductMixin(
+  ContentMixin<CartItemAvailabilityOptions>(LitElement)
+) {
+  static styles = availabilityStyles;
+
+  protected status = computed(() => {
+    const { isNeverOutOfStock, quantity, availability } =
+      this.$product()?.availability ?? {};
+    const { threshold: threshold = Infinity } = this.$options();
```

Comment:
`const { threshold = Infinity } = this.$options();`
and whether it is reasonable to use Infinity as default value?